migtests/scripts/live-migration-run-test.sh

```diff
@@ -52,12 +52,18 @@ main() {
 	step "Initialise source database."
 	./init-db
 
-	step "Grant source database user permissions"
+	step "Grant source database user permissions for live migration"
 	if [ "${SOURCE_DB_TYPE}" = "oracle" ]
 	then
 		grant_permissions_for_live_migration_oracle ${ORACLE_CDB_NAME} ${SOURCE_DB_NAME}
-	else
-		grant_permissions ${SOURCE_DB_NAME} ${SOURCE_DB_TYPE} ${SOURCE_DB_SCHEMA}
+	elif [ "${SOURCE_DB_TYPE}" = "postgresql" ]
+	then
+		set_replica_identity
```

Comment:
Probably should create a function grant_permission_live so that you can incorporate all database types in that function instead of writing the if-else logic here.